packages/flutter/test/painting/text_style_test.dart

```diff
@@ -29,6 +30,9 @@ class _DartUiTextStyleToStringMatcher extends Matcher {
     _propertyToString('letterSpacing', textStyle.letterSpacing),
     _propertyToString('wordSpacing', textStyle.wordSpacing),
     _propertyToString('height', textStyle.height),
+    // TODO(LongCatIsLooong): web support for
+    // https://github.com/flutter/flutter/issues/72521
```

Comment:
Did you or MH come up with a plan on how to do this?  Hopefully it will now map closely to what CSS does with leading and `line-height`.

Reply:
Oh and is it documented anywhere that leadingDistribution has no effect on web right now?

Reply:
Just noticed that you wrote that you plan to update the documentation later.

Reply:
I think the canvaskit backend is now the default. Not sure how this can be done with the css backend. I'll come back to the documentation when I start working on the web part.

Reply:
Sounds good.  Well it should be much easier on the canvaskit backend, and it should end up working closely to how CSS does it anyway since this half leading approach came from the CSS spec as I understand it.